wiki/Tournaments/otst!/2019/en.md

```diff
@@ -138,13 +136,12 @@ The osu! TV Size Tournament 2019 is run by various osu! community members.
 
 ### Scheduling
 
-- **All times in the match schedules are in UTC**
-
-1. Scheduling will be based on team timezones to avoid as many no-shows as possible.
-2. If your team wants to reschedule a match, please have yours and the opposing team agree on the new time and have either team captain submit proof of agreement in the Discord server's `#reschedule` text channel by **Thursday 23:59 UTC**.
-3. During the group stage, a 3v3 match could be arranged if some teammates cannot make it. The other team must be notified and agree to it too.
+1. **All times in the match schedules are in UTC**.
```

Comment:
this doesn't belong in the list because it has nothing to do with the actual tournament, it's just a note about the wiki. it should be in its original place but without the list (and probably italics instead of bold?)

Reply:
also the same thing about the stop that i commented on other part

Reply:
or maybe remove too, since everyone can notice it in the match schedules later on?